IFC4x3/Properties/s/SlopeAngle_1e_6tKMk1BoO7cfc4MFmey/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="SlopeAngle_1e_6tKMk1BoO7cfc4MFmey" Name="SlopeAngle" UniqueId="68f86dd4-5ae0-4bc9-81e6-a661163f0a3c" PrimaryDataType="IfcPlaneAngleMeasure" />
```

Comment:
This slope angle is relative to what?

Reply:
to horizontal (90° is avertical slope). Common terminology in civil engineering